web/app/themes/wordpress-scaffold/lib/Grrr/Theme/Config.php

```diff
@@ -0,0 +1,66 @@
+<?php namespace Grrr\Theme;
+
+final class Config {
+
+    /**
+     * Post type registration names and their class names.
+     */
+    const POST_TYPES = [
+        'comment' => 'Comment',
```

Comment:
Having to add these twice still feels kind of stupid. Maybe we can leverage the power of something like `UtilsLoader` to automatically fetch the class name and type? Ignoring all `PostType*.php` files an simply invoking the class and fetching the `$type` would do it...